pandas/core/generic.py

```diff
@@ -3525,46 +3525,45 @@ def filter(self, items=None, like=None, regex=None, axis=None):
         Parameters
         ----------
         items : list-like
-            List of info axis to restrict to (must not all be present)
+            List of info axis to restrict to (must not all be present).
```

Comment:
Hmm I don't think `info axis` is well known. How about "List of labels from `axis` to restrict to."

Reply:
And maybe replace all other "info axis" with \`axis\`